jetbrains-core-gui/tst/software/aws/toolkits/jetbrains/EmptyProjectTestCase.kt

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains
+
+import com.intellij.testGuiFramework.impl.GuiTestCase
+import com.intellij.testGuiFramework.util.scenarios.newProjectDialogModel
+import org.junit.Before
+import software.aws.toolkits.jetbrains.fixtures.createEmptyProject
+import java.nio.file.Path
+import java.nio.file.Paths
+
+abstract class EmptyProjectTestCase : GuiTestCase() {
+
+    protected val testDataPath: Path = Paths.get(System.getProperty("testDataPath"))
+
+    @Before
+    fun createEmptyProject() {
+        welcomeFrame {
+            createNewProject()
```

Comment:
Did the openProject(folder) not work?

Reply:
It did - but then it required going down the road of committing `.idea` files - and they might change per version, had path specific pieces that are going to differ per machine etc - it seemed like a treadmill we didn't need to be on. This was a much simpler implementation than the previous one and it seemed like a better way to go.

Reply:
Okay, if its stable seems fine. Wonder if we can do something ourself later that doesnt need .idea so its faster but lets worry about that when UI tests become slow to run